pkg/utils/cni.go

```diff
@@ -0,0 +1,42 @@
+/*
+Portions Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"github.com/Azure/karpenter-provider-azure/pkg/consts"
+)
+
+const (
+	// defaultKubernetesMaxPodsAzure is the maximum number of pods to run on a node for Azure CNI.
+	defaultKubernetesMaxPodsAzure = 250
+	// defaultKubernetesMaxPodsKubenet is the maximum number of pods to run on a node for Kubenet.
+	defaultKubernetesMaxPodsKubenet = 100
```

Comment:
Why wouldn't we remove this altogether?

Reply:
I wasn't certain if we would ever support kubelet. 

It could make some code much simpler if we didn't have to also consider network-plugin. Although maybe by just haivng this someone may get the impression that we do support it.

Reply:
One point here -- not gone forever. We always have this code in our git history, so we can re-add and resolve any library/API conflicts quickly if they arise

Reply:
Yeah removed all of this garbage.